service/history/workflowResetter.go

```diff
@@ -583,7 +583,7 @@ func (r *workflowResetterImpl) terminateWorkflow(
 		eventBatchFirstEventID,
 		terminateReason,
 		nil,
-		consts.IdentityHistoryService,
+		"resetter",
```

Comment:
Instead of hardcoding the same string value in multiple places, please define a constant for it